docs/scripts/util/index.ts

```diff
@@ -1,4 +1,2 @@
 export { getAllTypesData } from './getAllTypesData';
 export { getCatalog } from './getCatalog';
-export { getFrontmatter } from './getFrontmatter';
```

Comment:
frontmatter can be get directly from the page.mdx files, so no need to add them to the props-table-data.json